tests/cli/test_cli_parser.py

```diff
@@ -266,20 +272,12 @@ def test_cli_parser_executors(self, executor, expected_args):
             ) as stderr:
                 reload(cli_parser)
                 parser = cli_parser.get_parser()
-                with pytest.raises(SystemExit) as e:
+                with pytest.raises(SystemExit) as e:  # running the help command exits, so we prevent that
                     parser.parse_args([expected_arg, "--help"])
-                    assert e.value.code == 0
```

Comment:
this assert was never executed because it was in the block that captures the exception